test/com/moclojer/resources/moclojer.yml

```diff
@@ -12,7 +12,7 @@
       # :username placeholder
       body: >
         {
-          "hello": "{{path-params.username}}!"
+          "hello": "{{path.username}}!"
```

Comment:
@matheusfrancisco ao invés de quebrar a compatibilidade, não podemos injetar o conteúdo do `path` no `path-params`?

Reply:
might be a good idea.